modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/incubus_draft.dm

```diff
@@ -135,26 +136,32 @@
 	return ..()
 
 /datum/reagent/drug/aphrodisiac/incubus_draft/overdose_effects(mob/living/carbon/human/exposed_mob)
+	// Check if overdosing on incubus draft and succubus milk simultaneously, to prevent chat spam
+	var double_dosing = FALSE
+	for(var/r in exposed_mob.reagents.reagent_list)
+		var/datum/reagent/reagent = r
+		if(reagent.name == "succubus milk" && reagent.overdosed)
+			double_dosing = TRUE
+
+	// Begin cock growth
 	if(exposed_mob.client?.prefs?.read_preference(/datum/preference/toggle/erp/penis_enlargement))
+		// Start making new genitals if prefs allow it and if there isn't already one there
 		if(!exposed_mob.getorganslot(ORGAN_SLOT_PENIS) && exposed_mob.client?.prefs?.read_preference(/datum/preference/toggle/erp/new_genitalia_growth))
 			var/list/data = species_to_penis[exposed_mob.dna.species.id]
 			if(!data)
 				data = species_to_penis[SPECIES_HUMAN]
-
 			exposed_mob.dna.features["penis_sheath"] = data["sheath"]
-			exposed_mob.dna.mutant_bodyparts[ORGAN_SLOT_PENIS][MUTANT_INDEX_NAME] = data["index_name"]
+			exposed_mob.dna.mutant_bodyparts[ORGAN_SLOT_PENIS][MUTANT_INDEX_NAME] = data["mutant_index"]
```

Comment:
what's the need for this cahnge

Reply:
That was the bug that was causing the penis to not grow in the first place. 	

```
var/static/list/species_to_penis = list(
		SPECIES_HUMAN = list(
			"sheath" = SHEATH_NONE,
			"mutant_index" = "Human",
			"balls" = "Pair"
		),
```

It should be 'mutant_index', it was setting it to nothing before and so there would be no sprite or anything on examine.